templates/zerver/help/pin-a-stream.md

```diff
@@ -1,8 +1,8 @@
 # Pin a stream
 
-Pinning a stream moves it to the top of the left sidebar. We recommend
-pinning important streams if you have more streams than fit in the left
-sidebar.
+Pinning a stream moves it to the top of your streams list in the left sidebar
+under the section labeled **Pinned**. We recommend pinning important streams
+if you have more streams than fit in the left sidebar.
```

Comment:
How about something like this?

> Pinning a stream moves it to the **Pinned** section at the top of your streams list in the left sidebar.

Reply:
> We recommend pinning important streams
if you have more streams than fit in the left sidebar.

This doesn't match our style for not using "we" in the help center, so how about something like this instead?

> You can pin the streams you pay close attention to, which is especially handy if you are subscribed to a large number of streams.